source/mainnet/net/guides/overview-desktop.rst

```diff
@@ -7,57 +7,86 @@ Set up the Desktop Wallet
+Before you start, make sure you’ve completed the following steps:
 
-Step 2: Set up the LEDGER device
-================================
+- You’ve set up the `LEDGER Nano S <https://support.ledger.com/hc/en-us/articles/360000613793-Set-up-your-Ledger-Nano-S?docs=true>`_, `LEDGER Nano S Plus <https://support.ledger.com/hc/en-us/articles/4416927988625-Set-up-your-Ledger-Nano-S-Plus-?docs=true>`_, or the `LEDGER Nano X <https://support.ledger.com/hc/en-us/articles/360018784134-Set-up-your-Ledger-Nano-X?docs=true>`_ and you have saved your 24-word recovery phrase generated on the LEDGER securely. This is the only backup of your private keys, and you need it to access you accounts if you lose your LEDGER device.
 
-The Desktop Wallet requires that you store your keys on a LEDGER device. This is to ensure that your private account keys are kept secure. To be able to use the LEDGER device with the Desktop Wallet, you must install the Concordium LEDGER App on the hardware wallet. See :ref:`Install the Ledger App guide<install-ledger-app>`.
+- You’ve downloaded `Ledger Live <https://www.ledger.com/ledger-live/download>`_.
 
-Step 3: Set up the Concordium Desktop Wallet
-============================================
+.. dropdown:: Step 1: Set up the node
 
-You'll need to install and set up the Desktop Wallet to create and manage identities and accounts and add a baker.
+    The Desktop Wallet must be connected to a running **trusted** node on the Concordium blockchain. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and submit transactions. A default Virtual Hive node (concordiumwalletnode.com) is provided. But you can run a node yourself using :ref:`Windows<run-node-windows>`, :ref:`macOS <run-node-macos>`, and :ref:`with Docker <run-a-node>` or :ref:`a Debian package <run-node-ubuntu>` on Linux. You can also have a third-party provider run a node for you.
 
-.. Note::
-    Without CCD you can't submit transactions on the Concordium blockchain. This includes creating multi-signature accounts and creating baker transactions. However, you can create identities, accounts that only require one signature, and you can add account addresses to your address book.
+.. dropdown:: Step 2: Install the Desktop Wallet
 
-    You can buy CCD on many exchanges. If you are running on testnet, you can request CCD for testing using a button in the wallet.
+    #. :ref:`Download <downloads>` the latest version of the Desktop Wallet.
```

Comment:
I guess this could refer directly to the Desktop wallet topic on that page, e.g.:
```http://127.0.0.1:8000/net/installation/downloads.html#concordium-desktop-wallet```